esti/auth_test.go

```diff
@@ -224,3 +237,19 @@ func mergeAuthTest(t *testing.T, cli *apigen.ClientWithResponses, ctx context.Co
 
 	return client.MergeIntoBranchWithResponse(ctx, repo, branch, mainBranch, apigen.MergeIntoBranchJSONRequestBody{})
 }
+
+func mapGroupNamesToIDs(t *testing.T, ctx context.Context, groups []string) (map[string]string, []string) {
+	groupIDs := make([]string, len(groups))
+	mapGroupNameToID := make(map[string]string)
+
+	// get group list
+	resListGroups, err := client.ListGroupsWithResponse(ctx, &apigen.ListGroupsParams{Amount: apiutil.Ptr(apigen.PaginationAmount(-1))})
+	require.NoError(t, err, "unexpectedly failed while listing groups")
+	for _, group := range resListGroups.JSON200.Results {
+		mapGroupNameToID[*group.Name] = group.Id
+		if slices.Contains(groups, *group.Name) {
+			groupIDs = append(groupIDs, group.Id)
+		}
```

Comment:
why do you make this difference?
shouldn't the mapping also include only groups from the `groups` argument?

Reply:
An unlisted group is used [here](https://github.com/treeverse/lakeFS/pull/7326/files/6bd2ea1eba7538d6774d6b5241497c19e8b45c5c#diff-bf5383c0eb6d7a5da64042d91af722b2f13e9198624e13d3713c50c132feac61R104) line